src/com/couchbase/tools/performer/BuildPerformer.groovy

```diff
@@ -164,11 +166,11 @@ class BuildPerformer {
                 } else if (sdkRaw == "go") {
                     BuildDockerGoPerformer.build(env, dir, vers, imageName, onlySource)
                 } else if (sdkRaw == "python") {
-                    BuildDockerPythonPerformer.build(env, dir, vers, imageName, onlySource)
+                    BuildDockerPythonPerformer.build(env, dir, vers, imageName, onlySource, dockerBuildArgs)
```

Comment:
Should we explicitly reject the arg if it's used with an SDK that doesn't yet support it?

Reply:
Done